packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -352,13 +354,19 @@ export function useAutocomplete(props: AriaAutocompleteOptions, state: Autocompl
     }
   };
 
-  return {
-    textFieldProps: {
-      value: state.inputValue,
-      onChange,
+  // Only apply the autocomplete specific behaviors if the collection component wrapped by it is actually
+  // being filtered/allows filtering by the Autocomplete.
+  let textFieldProps = {
+    value: state.inputValue,
+    onChange
+  } as AriaTextFieldProps<HTMLInputElement>;
+
+  if (collectionId) {
+    textFieldProps = {
+      ...textFieldProps,
       onKeyDown,
       autoComplete: 'off',
-      'aria-haspopup': 'listbox',
+      'aria-haspopup': collectionId ? 'listbox' : undefined,
```

Comment:
just noticed we set this on autocompletes which don't have a "popup" in the usual sense. I assume that is correct behaviour though?
See first example on the docs for Autocomplete

Reply:
I think we might be able to get away with removing that attribute (think it came from useComboBox), but would need to test AT. Partially related to https://github.com/orgs/adobe/projects/19/views/4?sliceBy%5Bvalue%5D=Autocomplete&pane=issue&itemId=120817364, I'll revisit it as part of that work